### ISSUE TITLE
Extract Command Extracts Nothing

### ISSUE THREAD

**Issue by fny** [lingui/js-lingui#828]
I upgraded from 3.0.0 to 3.03, and lingui extract is broken. The extract command shows there are zero updates.

I check the prior versions, 3.0.1 and 3.0.2 work fine.

**Comment by semoal** [comment]
Looks like a regression with this Tomas change https://github.com/lingui/js-lingui/pull/803

Can you share your lingui.config pls?

**Comment by semoal** [comment]
Yes, the problem is the mandatory ending slash to know that is a directory. This bug is already fixed on main and will be released in the next cycle (Tomáš spoke about the next tuesday)

If you change this defualt configuration:

```js
"catalogs": [{
   "path": "<rootDir>/locale/{locale}/messages",
   "include": ["<rootDir>"],
   "exclude": ["**/node_modules/**"]
 }],
```

to this:

```js
"catalogs": [{
   "path": "<rootDir>/locale/{locale}/messages",
   "include": ["<rootDir>/"], // this / is mandatory
   "exclude": ["**/node_modules/**"]
 }],
```

Will be fixed. It's a temporal workaround because it'll be fixed next tuesday, you can use meanwhile the 3.0.2 👍🏻

**Comment by semoal** [comment]
Released 3.1.0 should be fixed, feel free to reopen
